manifests/config.pp

```diff
@@ -206,6 +206,7 @@
 
       if $foreman::ipa_manage_sssd {
         $sssd = pick(fact('foreman_sssd'), {})
+        $sssd_default_domain_suffix = join(unique(pick($sssd['default_domain_suffix'], []) + [$::domain]), ', ')
```

Comment:
The use of `$::X` is deprecated. Instead, we now use `$facts['domain']`.
```suggestion
        $sssd_default_domain_suffix = join(unique(pick($sssd['default_domain_suffix'], []) + [$facts['domain']]), ', ')
```

This implies `default_domain_suffix` is an array. Is that correct?

Additionally, the fact is not retrieved from sssd (see https://github.com/theforeman/puppet-foreman/blob/master/lib/facter/sssd.rb) so `$sssd['default_domain_suffix']` is always empty.

That means right now it's effectively:
```suggestion
        $sssd_default_domain_suffix = $facts['domain']
```

Reply:
And sincere apologies for the delay in replying.

Reply:
> And sincere apologies for the delay in replying.

Not at all - I actually had PTO on Thursday and Friday so to me there was no delay.